services/QuillLMS/client/app/bundles/Staff/components/comprehension/regexRules/regexRulesIndex.tsx

```diff
@@ -49,6 +49,15 @@ const RegexRulesIndex: React.FC<RouteComponentProps<ActivityRouteProps>> = ({ ma
     queryFn: fetchRules
   });
 
+  function handleReorder(sortInfo) {
+    const idsInOrder = sortInfo.map(item => item.key)
+    updateRuleOrders(idsInOrder).then((response) => {
+      queryCache.refetchQueries([`rules-${activityId}-${RULES_BASED_1}`])
+      queryCache.refetchQueries([`rules-${activityId}-${RULES_BASED_2}`])
+      queryCache.refetchQueries([`rules-${activityId}-${RULES_BASED_3}`])
```

Comment:
I'm not sure if this will work anymore? Either way, you can just put `queryCache.clear()` here

Reply:
It did, I tested it! Won't `queryCache.clear()` wipe the whole cache?

Reply:
oh, awesome! I wonder why it stopped working in some places... Yes, it'll clear the whole cache.  When we eventually update the package, I think the docs suggest that that is the preferred way to update since `refetchQueries` is already phased out in newer versions